.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.8.7
+  - 1.17.5
```

Comment:
CI seems broken, I will open a PR to fix it first if you want. SGTY @thaJeztah?

Reply:
Yeah, although don't spend _too much_ time on it; we really need to reduce use of this dependency (it's good to know if things are not broken of course)

Reply:
Yeah that's a huge jump 1.8 > 1.17, better to do that in a follow-up :sweat_smile:

Reply:
I guess I'm just being very opportunistic and putting all my trust in Go's backwards compatibility promise 😹
And, to be fair, this has already been built in Moby using all mort recent versions of Go...